addons/spreadsheet/static/src/pivot/plugins/pivot_ui_plugin.js

```diff
@@ -195,20 +195,32 @@ export class PivotUIPlugin extends spreadsheet.UIPlugin {
         return undefined;
     }
 
+    /**
+     * Returns the first pivot function of formula with its evaluated arguments.
+     *
+     * @param {string} formula
+     * @returns { { functionName: string, args: Array<string> } | undefined }
+     */
     getFirstPivotFunction(formula) {
-        const pivotFunction = getFirstPivotFunction(formula);
-        if (!pivotFunction) {
-            return undefined;
-        }
-        const { functionName, args } = pivotFunction;
-        const evaluatedArgs = args.map((argAst) => {
-            if (argAst.type == "EMPTY") {
+        try {
+            const pivotFunction = getFirstPivotFunction(formula);
```

Comment:
Do'nt we have the exact same problem for all the places where we use `evaluateFormula` in Odoo (SEE_RECORD_LIST/getListIdFromPosition/...) ? Maybe we should also check them.

Reply:
We do, I actually tested it, don't  know how I missed it.